Connectors/src/CosmosDb/Controllers/HomeController.cs

```diff
@@ -1,38 +1,45 @@
-﻿using Microsoft.AspNetCore.Mvc;
-using System;
-using System.Threading.Tasks;
+﻿using System.Diagnostics;
+using CosmosDb.Data;
+using CosmosDb.Models;
+using Microsoft.AspNetCore.Mvc;
 
-namespace CosmosDb
+namespace CosmosDb.Controllers;
+
+public class HomeController : Controller
 {
-    public class HomeController : Controller
+    private readonly ILogger<HomeController> _logger;
+    private readonly ICosmosDbService _cosmosDbService;
+
+    public HomeController(ILogger<HomeController> logger, ICosmosDbService cosmosDbService)
     {
-        private readonly ICosmosDbService _cosmosDbService;
+        _logger = logger;
+        _cosmosDbService = cosmosDbService;
+    }
 
-        public HomeController(ICosmosDbService cosmosDbService)
-        {
-            _cosmosDbService = cosmosDbService;
-        }
+    public async Task<IActionResult> Index(CancellationToken cancellationToken)
+    {
+        // Steeltoe: Fetch objects from CosmosDB container.
+        var model = new CosmosDbViewModel();
 
-        // GET: /<controller>/
-        public async Task<IActionResult> Index()
+        await foreach (SampleObject sampleObject in _cosmosDbService.GetAllAsync(cancellationToken))
         {
-            var testData = await _cosmosDbService.GetTestDataAsync("SELECT * FROM c");
-
-            return View(testData);
+            model.SampleObjects.Add(sampleObject);
         }
 
-        public async Task<IActionResult> AddData()
```

Comment:
Does add/delete not work or are we saying we don't need to demo here?

Reply:
The latter. None of the db-connector samples contain that.